packages/devtools_app/lib/src/shared/release_notes/release_notes.dart

```diff
@@ -137,7 +155,12 @@ class ReleaseNotes extends AnimatedWidget {
         child: Column(
           children: [
             AreaPaneHeader(
-              title: const Text('What\'s new in DevTools?'),
+              title: const Expanded(
```

Comment:
do we need this to be in an `Expanded`? The actions that are added to the same row as `title` in `AreaPaneHeader` will be placed in `Expanded` widgets, and generally you only want one `Expanded` per `Row` or `Column`. Also, it is best practice to place `Expanded` and `Flexible` widgets directly in the `Row` or `Column` they'll be used in instead of defined somewhere outside of that context. This makes it clearer to understand how a flex layout is constructed.

Reply:
I will look into this to verify :)

Reply:
So in this case the ellipses won't show on the title unless that expansion is there. I will take a look to see if there is a  way that I can Rectify this issue in a way that satisfies the best practices you have listed